030-Data-types/030-data-model.mdx

```diff
@@ -7,6 +7,33 @@ slug: concepts/data-model
 published: true
 ---
 
+Xata builds on top of Postgres and thus supports the [default data types](https://www.postgresql.org/docs/current/datatype.html) inherit in that platform. Xata additionally adds to that list with unique types such as `file`, `filearray` and `vector` to handle platform features like [file attachements](/docs/sdk/file-attachments) and [vector searching](/docs/sdk/vector-search). In addition, there are some added types to simplify validation of common `text` patterns such as `email` and `multiple`. Usage of these types is ultimately optional if you wish to use Xata only as a Postgres host. If you do decide to utilize them (which is the default way Xata operates through our UI), it is required that you interact with these types through the Xata SDKs.
```

Comment:
Maybe `text` is not a good example, as it's ambiguous. In Xata we have the limitation of 1000 chars and in PG there is "none". I added a bit about this limitation in this https://github.com/xataio/mdx-docs/pull/281/commits/9565399fa9a5640479039042baa5a797ff4c3a04

Reply:
Added a link to this issue when discussing limitations in the section below. Also cleaned up the language.